moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -80,33 +82,23 @@ export const resolveHasSubprojects = (array) => {
   return "No";
 };
 
-const filterComponentFullNames = (value) => {
-  if (!value.components) {
-    return "";
```

Comment:
nice! this reminded me to check what everything looks like when a project has no components, personnel, etc. I see blank cells as expected.